src/api/tsIntegrationTest.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * You may safely remove this file once other typescript exists.
+ * This is simply a test for how to integrate new ts functionality into existing code.
+ */
+export function thisIsATest(input: number | undefined): number {
+  if (input === undefined) {
+    return -1
+  }
+  return input + 2
+}
```

Comment:
Not a big deal, but my vote is to either not check this in or move it to an examples folder

Reply:
👍 sounds good, i'll remove it. we'll have real-life examples sooon